pagopa/io-app
# chore: [IOBP-555] Add payment outcome screen for the outcome 17 and 15

## Short description
This PR adds the new payment outcome screens for the outcome number `15` and `17`

## List of changes proposed in this pull request
- Added locales for the title and subtitle about the two new outcomes;
- Mapped outcome 15 as `METHOD_NOT_ENABLED` and `WAITING_CONFIRMATION_EMAIL` from the `WalletPaymentOutcomeEnum`
- Added additional cases to the switch located inside the`WalletPaymentOutcomeScreen` setting the title, subtitle, and pictogram to show.

## How to test
- Check out this PR from the io-dev-app-server: https://github.com/pagopa/io-dev-api-server/pull/351
- Start the payment process until you reach the webview with the outcome list
- Then choose outcome number 15 or 17 to check out the new outcome screens

## Preview
https://github.com/pagopa/io-app/assets/34343582/bd67f00f-37ae-4b0f-b33d-944884fe88a3
